PowerPointLabs/PowerPointLabs/TextCollection/EffectsLabText.cs

```diff
@@ -75,6 +75,10 @@ internal static class EffectsLabText
             "Recolor everything in the slide except for the selected shapes.\n\n" +
             "To perform this action, select the shape(s) or picture(s) to keep, then click this button.";
 
+        public const string MagnifyActionHandlerReminderText = "Please select an area to magnify.";
+        public const string MagnifyOnlyOneAreaReminderText = "Only one magnify area is allowed.";
+        public const string SelectAtLeastOneShapeReminderText = "Please select at least 1 shape.";
```

Comment:
Why did you decide to call these texts "ReminderText"?
And if they are really Reminders, shouldn't it be ReminderMagnifyOneArea. I don't think they are reminders though. Look like error messages to me.

Reply:
I will change this asap

Reply:
I have changed from ReminderText to Error messages